### PR TITLE
Standardize the way questions are rendered

### PR DESCRIPTION
**Pull request by natsid**
### Description
Previously each question type's renderer was rendering its own question text, help text, and errors. This puts all of that logic in a common `ApplicantQuestionRenderer#renderInternal` method.

Based on Figma mocks. Also slightly improve Block and question error rendering.

### Screenshots
Screen Shot 2021-05-14 at 4 19 36 PM
Screen Shot 2021-05-14 at 3 22 07 PM
Screen Shot 2021-05-14 at 3 51 10 PM
Screen Shot 2021-05-14 at 4 00 57 PM

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Works toward #1016

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRenderer.java [hunk 1]
```diff
@@ -1,8 +1,67 @@
...
+import j2html.tags.ContainerTag;
 import j2html.tags.Tag;
+import play.i18n.Messages;
+import services.applicant.question.ApplicantQuestion;
+import views.BaseHtmlView;
+import views.style.ApplicantStyles;
+import views.style.ReferenceClasses;
+import views.style.Styles;
+
+public abstract class ApplicantQuestionRenderer {
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRenderer.java:15]: praise: love that this is an abstract class!

File: universal-application-tool-0.0.1/app/views/style/BaseStyles.java [hunk 2]
```diff
@@ -11,25 +11,28 @@
   // CiviForm color classes
   /////////////////////////////////////////////////////////////////////////////////////////////////
 
+  public static final String BG_CIVIFORM_WHITE = "bg-civiformWhite";
+
   public static final String BG_SEATTLE_BLUE = "bg-seattleBlue";
-  public static final String BG_CIVIFORM_BLUE_60 = "bg-civiformBlue-60";
-  public static final String BG_CIVIFORM_BLUE_80 = "bg-civiformBlue-80";
-  public static final String BG_CIVIFORM_BLUE_100 = "bg-civiformBlue-100";
+  public static final String TEXT_SEATTLE_BLUE = "text-seattleBlue";
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/views/style/BaseStyles.java:17]: question (non-blocking): Out of curiosity, is there a plan to move these to be more general, i.e. another city can use their own color scheme?

**Review by carolinedanzi — APPROVED** [review]
Really nice work!

File: universal-application-tool-0.0.1/app/views/style/BaseStyles.java (hunk 2 above)
Comment (natsid) [reply universal-application-tool-0.0.1/app/views/style/BaseStyles.java:17]: Yes, I really hope so! It's something we haven't talked about yet, but need to.

File: universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRenderer.java (hunk 1 above)
Comment (natsid) [reply universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRenderer.java:15]: wohoo! 😃

File: universal-application-tool-0.0.1/app/views/style/BaseStyles.java (hunk 2 above)
Comment (natsid) [reply universal-application-tool-0.0.1/app/views/style/BaseStyles.java:17]: I figured I'd just add the Seattle color scheme in this hard-coded way for now for the pilot. I'd love to somehow make this configurable.

**Pull request merged**